.github/workflows/ci.yml

```diff
@@ -297,6 +299,19 @@ jobs:
       shell: bash
       run: cross test --target ${{ matrix.target }} --manifest-path "aws/rust-runtime/Cargo.toml" ${{ matrix.test_aws_exclude }} --workspace
 
+  # compile time benchmark
```

Comment:
Gotcha.

Reply:
I initially thought about using AWS batch, since you can have better control over the hardware, thus making results more re-producible and intuitive to understand.

Since I don't have access to your AWS account, I decided to take this route. 
Just FYI.